doc/release-notes/release-7.sgml.in

```diff
@@ -39,23 +38,28 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 <sect1>Cache Manager changes<label id="mgr">
 <p>For more information about the Cache Manager feature, see <url url="https://wiki.squid-cache.org/Features/CacheManager/Index" name="wiki">.
 
-<p>
-<descrip>
-	<tag>cache_object URI</tag>
-	<p>Cache manager is no longer accessible by URIs with cache_object scheme.
+<p>In order to reduce workload on the Squid development team we have chosen to stop
+   providing several tools related to Cache Manager which have previously been
+   bundled with Squid.
 
-    <tag>cachemgr</tag>
-    <p>The 'cachemgr' tool is no longer supported and distributed with squid.
-    A plethora of tools, such as curl, wget, or any web browser,
-    can be used in its place
+<sect2>Removal of the <em>squidclient</em> tool.
+<p>Alternative tools such as <em>curl</em> or <em>wget</em> provide equivalent
+   features and are more familiar to administrators these days.
 
-	<tag>non_peers</tag>
-	<p>Removed the <em>mgr:non_peers</em> report. Squid still ignores
-	unexpected ICP responses but no longer remembers the details that comprised
-	the removed report. The senders of these ICP messages are still reported to
-	cache.log at debugging level 1 (with an exponential backoff).
+<sect2>Removal of the <em>cachemgr.CGI</em> tool.
+<p>Access to the Cache Manager API is available by sending HTTP(S) requests
+   directly to Squid with the URL-path prefix <em>/squid-internal-mgr/</em>.
+   A plethora of tools, such as curl, wget, or any web browser, can be used
+   in its place
```

Comment:
Technically, "it" refers to Cache Manager API or URL-path prefix in this context:

```suggestion
   instead of cachemgr.cgi.
```